### DISCUSSION TITLE
Is this repo active?

### DISCUSSION THREAD

**Discussion by AhmadiRamin** [pnp/powerautomate-samples#26] [General]
Hi,

I can see there are some pull requests from months ago, wondering if this repo is active and is checked regularly?

**Comment by aprildunnam** [comment]
Yes, the repo is very much still active. If you look, you'll see several recent samples added. Each PR is thoroughly reviewed and tested before processing which takes time. PR processing can be delayed for multiple reasons like issues with the submission requiring changes or illnesses/vacation/scheduling conflicts with the volunteer maintenance team. 

I would highly encourage you to participate in the M365 Community General Dev calls that are bi-weekly on Thursdays.  We highlight every contribution to this repo on that call even if it hasn't been processed yet. I believe your sample was highlighted on the call yesterday.

**Comment by AhmadiRamin** [comment]
Thank you April, I usually participate in the M365 community general dev calls, but didn't have a chance to join this week, will watch the recordings :)
